src/test/java/io/stargate/sgv2/jsonapi/api/v1/CreateCollectionIntegrationTest.java

```diff
@@ -47,6 +48,152 @@ public void happyPath() {
           .statusCode(200)
           .body("status.ok", is(1));
     }
+
+    @Test
+    public void duplicateCollectionName() {
```

Comment:
I think this test method is getting long and should be split into multiple separate test methods, to keep it more readable.